opendps/dps-model.h

```diff
@@ -3,7 +3,41 @@
 
 // if no other DPSxxxx model is specified, we will assume DPS5005
 
-// #define DPS5015 
+#define DPS5015 
```

Comment:
This would surprise current 5005 users. I could go with the makefile complaining unless the model was specified at compile time.